packages/codemod/transformations/core/v2-to-v3/tooltip-component-migration.ts

```diff
@@ -20,6 +21,7 @@ function transform({ j, root }: TransformationContext) {
 
   elements.forEach(elementPath => {
     removeProp(j, elementPath, "withMaxWidth");
+    removeProp(j, elementPath, "showTooltipOnlyOnTriggerElement");
```

Comment:
```suggestion
    removeProp(j, elementPath, "withMaxWidth", "showTooltipOnlyOnTriggerElement");
```

should also work, should save an iteration of `O(n^2)` (probably negligible, so for your consideration)